### ISSUE TITLE
Clicking twice on the 'overlay' button makes it impossible to close the overlay 

### ISSUE THREAD

**Issue by RuneBarrett** [notablackbear/poe_qolV2#8]
As the title says, the overlay window showing the count can't be closed (using the X button) if i press the 'overlay' button more than once.

**Comment by notablackbear** [comment]
Yes I encountered the same behavior. This is a low priority bug since you should only need to open the overlay once anyway. I'll leave this issue open for now.

**Comment by RuneBarrett** [comment]
Agreed, this is only a qol thing. No pun intended ;)
